torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4353,6 +4377,100 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            op=torch.mode,
            dtypes=all_types_and(torch.float16, torch.bfloat16, torch.bool),
            sample_inputs_func=sample_inputs_mode,),
+    UnaryUfuncInfo('mvlgamma',
+                   ref=reference_mvlgamma if TEST_SCIPY else _NOTHING,
+                   variant_test_name='mvlgamma_p_1',
+                   domain=(1e-4, float('inf')),
+                   decorators=(precisionOverride({torch.float16: 5e-2}),),
+                   dtypes=floating_types(),
+                   dtypesIfCPU=floating_types(),
+                   dtypesIfCUDA=floating_types_and(torch.half),
+                   sample_inputs_func=sample_inputs_mvlgamma,
+                   supports_out=False,
+                   skips=(
+                       # outside domain values are hard error for mvlgamma op.
+                       SkipInfo('TestUnaryUfuncs', 'test_float_domains'),
+                   ),
+                   sample_kwargs=lambda device, dtype, input: ({'p': 1}, {'d': 1})),
+    UnaryUfuncInfo('mvlgamma',
```

Comment:
How different are the code paths or mathematics for the different p values of mvlgamma? Should we only OpInfo two values of `p`, for example?

Reply:
Codepath is same for different `p` and consists of functions which are already tested.
https://github.com/pytorch/pytorch/blob/77721ee318d6785010144aa4569efb98199e7162/aten/src/ATen/native/UnaryOps.cpp#L656-L669

However with increasing values of `p` the function grows faster (image below). Also note that since `scipy` computes in `double` for float16 input (while PyTorch in `float16`), we need to add skip for 
`SkipInfo('TestUnaryUfuncs', 'test_reference_numerics_hard', dtypes=(torch.float16,)),` for `p > 2`

With this in mind, I think we can add entries for `p={1, 3, 5}`.
Let me know what you feel about the same? (for now I have kept `p={1,2,3,4,5}` as before).
![Screenshot from 2021-04-28 13-57-27](https://user-images.githubusercontent.com/19503980/116373670-315a6580-a82b-11eb-9074-6a781b6fd530.png)

Reply:
1, 3, 5 sounds good; nice investigation, @kshitij12345